packages/api/src/command/webhook/webhook.ts

```diff
@@ -80,20 +86,23 @@ export const processRequest = async (
     });
   };
 
-  const payload = webhookRequest.payload;
+  const payload = webhookRequest.payload as Record<string, string>;
   try {
     const meta: WebhookMetadataPayload = {
       messageId: webhookRequest.id,
       when: dayjs(webhookRequest.createdAt).toISOString(),
```

Comment:
do we still need this and the timestamp in the header?

Reply:
ditto: keep this one